app/src/sharedTest/java/org/oppia/app/walkthrough/WalkthroughWelcomeFragmentTest.kt

```diff
@@ -41,7 +44,8 @@ import javax.inject.Singleton
 @RunWith(AndroidJUnit4::class)
 class WalkthroughWelcomeFragmentTest {
 
-  @Inject lateinit var profileTestHelper: ProfileTestHelper
+  @Inject
+  lateinit var profileTestHelper: ProfileTestHelper
```

Comment:
Annotate `@Inject` to single line.

Reply:
Ktlint is a bit different, it says to be on the next line. Also, Ben discuss the same in yesterday's meeting